### PR TITLE
Add mod classes for numeric input

### PR DESCRIPTION
**Pull request by pochretien**
### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/vapor/scss/controls/numeric-input.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        width: $numeric-input-2-digit-width;
+    }
+
+    .mod-max-3-digit {
+        width: $numeric-input-3-digit-width;
+    }
+
+    .mod-max-4-digit {
+        width: $numeric-input-4-digit-width;
+    }
```
Comment (GermainBergeron) [inline packages/vapor/scss/controls/numeric-input.scss]: I bet we could do a loop with a simple calculation (something like `padding + i * 10` )

File: packages/react-vapor/src/components/numericInput/NumericInputConnected.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface NumericInputOwnProps {
     min?: number;
     max?: number;
     invalidMessage?: string;
+    maxDigitWidth?: 2 | 3 | 4;
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/numericInput/NumericInputConnected.tsx]: Why start at 2? I'm wondering if we should set it to `number` instead

File: packages/react-vapor/src/components/numericInput/NumericInputConnected.tsx [hunk 3]
```diff
@@ -87,7 +88,14 @@ export class NumericInputConnected extends React.PureComponent<NumericInputProps
...
+                                'js-numeric-input',
+                                {
+                                    [`mod-max-${this.props.maxDigitWidth}-digit`]: _.isNumber(this.props.maxDigitWidth),
+                                },
+                                this.props.className,
+                                styles.numericInput
+                            )}
                             value={this.props.value}
                             onChange={this.onChange}
                         />
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/numericInput/NumericInputConnected.tsx]: no maxLength?

**Comment by pochretien** [comment]
Screen Shot 2019-10-18 at 4 04 32 PM

**Pull request merged**